Filter/Persister/SessionFilterPersister.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Filter\Persister;
+
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+/**
+ * This filter persister is storing filters in session.
+ * This is the default behavior.
+ *
+ * @author Yann Eugoné <eugone.yann@gmail.com>
+ */
+final class SessionFilterPersister implements FilterPersisterInterface
```

Comment:
@jlamur probably because https://ocramius.github.io/blog/when-to-declare-classes-final/
We are currently trying to close the API to avoid having to worry about extension BC-breaks, but also for reasons mentioned in this article: http://fabien.potencier.org/pragmatism-over-theory-protected-vs-private.html